src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,8 @@ public class CliSyntax {
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
-
```

Comment:
Are we keeping the `*/` prefix from AB3 or are we going to switch to using flags for fields?

Reply:
It seems that for now it's more convenient to stick with the previous rules. Later we can change them altogether if we decide to do so